src/main/java/Product.java

```diff
@@ -0,0 +1,36 @@
+import java.util.ArrayList;
+
+public class Product {
```

Comment:
Please add functional Javadoc explaining the intention/responsibility of a class